src/components/ReportActionItem/TaskPreview.js

```diff
@@ -149,5 +156,9 @@ export default compose(
             key: ({taskReportID}) => `${ONYXKEYS.COLLECTION.REPORT}${taskReportID}`,
             initialValue: {},
         },
+        policy: {
+            key: ({policyID}) => `${ONYXKEYS.COLLECTION.POLICY}${policyID || '0'}`,
```

Comment:
We don't need in this case to get the `ReportUtils.getRootParentReport(report);` first?

Reply:
Updated name of props. policyID props here was already policyID of root parent report

Reply:
@aldo-expensify Updated. please check again